src/features/alerts/components/NewAlertForm.tsx

```diff
@@ -0,0 +1,140 @@
+import { Button } from "@/src/components/ui/button";
+import * as z from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/src/components/ui/form";
+import { api } from "@/src/utils/api";
+import { useState } from "react";
+import { usePostHog } from "posthog-js/react";
+import { Input } from "@/src/components/ui/input";
+
+const formSchema = z.object({
+  name: z.string(),
+  alertThreshold: z.number(),
+  triggerWebhookUrl: z.string(),
```

Comment:
Can we validate here that the URL is valid? This would reduce the amounts of errors in case we alert and the url is invalid.